example-cropping/README.md

```diff
@@ -0,0 +1,779 @@
+              <Link href="#" variant="body2" onClick={()=>props.setCurrentPage(props.currentPage)}>
+                {"Don't have an account? Sign Up"}
+              </Link>
+            </Grid>
+          </Grid>
+        </form>
+      </div>
+    </Container>
+  );
+}
+
+export {Login}
+
+```
+
+example usage
+
+```js
+   <Login
+            appwrite={appwrite}
+            currentPage={currentPage}
+            getUserData={getUserData}
+            setCurrentPage={(currentPage) => setCurrentPage(!currentPage)}
+          />
+```
+it takes four props which is 
+
+- ```appwrite``` to connect to appwrite server using sdk
+- ```currentPage``` the state of login or sign up
+- ```setCurrentPage``` to switch between login and sign up
+- ```getUserData``` to check we already login or not
+
+here we implement login because in order for us to be able to crop an image we should login to our appwrite server and this component does exactly that.
+
+## Step 4 - Create UploadImage Component
+
+UploadImage component is used for uploading component through appwrite SDK so after we login in order for us to crop an image we should upload our image so that we can crop it later.
+
+```js
+import React, { useState } from 'react';
+import Button from "@material-ui/core/Button";
+function UploadImage(props) {
+  const [loading, setLoading] = useState(false);
+  const [uploadFile, setUploadFile] = useState(null);
+  const [error, setError] = useState(false);
+
+  let processUpload = async (props) => {
+    setLoading(true)
+    try {
+      let promise = await props.appwrite.storage.createFile(uploadFile, ['*'], ['*']);
+      setLoading(false)
+    } catch (e) {
+      setLoading(false)
+      setError(e.message)
+    }
+
+
+
+  }
+  let onFileChange = e => {
+    setUploadFile(e.target.files[0])
+  }
+
+  return (
+    <div>
+      <h1>Upload Image</h1>
+      {error && (<p>{error}</p>)}
+      <input type="file" onChange={onFileChange} />
+      <Button
+          
+          variant="contained"
+          color="primary"
+          disabled={loading} onClick={() => { processUpload(props) }}
+        >
+         Upload
+        </Button>
+     
+
+    </div>
+  )
+
+};
+
+export { UploadImage };
+```
+
+example usage
+
+```js
+  <UploadImage appwrite={appwrite} />
+```
+
+this only take one props which is appwrite for connecting to our server using appwrite sdk also if you notice this code
+
+```js
+await props.appwrite.storage.createFile(uploadFile, ['*'], ['*']);
+```
+
+is actually for uploading our file whether its image or but in this tutorial we will use image and this function actually take three parameters you can take a look the detailed explanation here https://appwrite.io/docs/client/storage#createFile
+
+## Step 5 - Create ListImage Component
+
+So after we done uploading our image we want to see all of our image that we upload right? so in order to do that we create a ```ListImage``` component so that we can show our image from the server
```

Comment:
```suggestion
So after we're done uploading our image we would want to see all of our images that we uploaded right? So in order to do that we create a ```ListImage``` component so that we can display our images stored on the server
```